io-engine/tests/nexus_replica_resize.rs

```diff
@@ -0,0 +1,324 @@
+pub mod common;
+
+use common::{
+    compose::{
+        rpc::v1::{
+            nexus::{ChildState, NexusState},
+            GrpcConnect,
+            SharedRpcHandle,
+        },
+        Binary,
+        Builder,
+    },
+    fio::{FioBuilder, FioJobBuilder},
+    nexus::NexusBuilder,
+    pool::PoolBuilder,
+    replica::ReplicaBuilder,
+};
+use std::time::Duration;
+
+use async_trait::async_trait;
+
+const POOL_SIZE: u64 = 500; // 500MiB
+const REPL_SIZE: u64 = 400; // 400MiB
+const EXPANDED_SIZE: u64 = 471859200; //450 MiB
+const DEFAULT_REPLICA_CNT: usize = 3;
+
+async fn compose_ms_nodes() -> io_engine_tests::compose::ComposeTest {
+    common::composer_init();
+
+    Builder::new()
+        .name("cargo-test")
+        .network("10.1.0.0/16")
+        .unwrap()
+        .add_container_bin(
+            "ms_nex_0",
+            Binary::from_dbg("io-engine").with_args(vec!["-l", "1,2"]),
```

Comment:
Doesn't need a vec I guess. Maybe an array would do?
```suggestion
            Binary::from_dbg("io-engine").with_args(&["-l", "1,2"]),
```

Reply:
> Doesn't need a vec I guess. Maybe an array would do?

This is consistent in all the existing io-engine cargo tests.